internal/rbd/controllerserver.go

```diff
@@ -732,7 +732,6 @@ func (cs *ControllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		}
 		return nil, err
 	}
-	defer rbdVol.Destroy()
```

Comment:
Might be better to remove the instance on line 720. I am not sure how Golang behaves, but the 1st instance might have been replaced by the return value on line 722, causing no `Destroy()` on the 2nd instance. Line 719 can be replaced by `var rbdVol *rbdVolume` as well.

Reply:
in the case of Error also need to Destroy the context. so I think we can place Destroy next to the struct.